src/Control/Lens/Internal/Iso.hs

```diff
@@ -22,16 +22,19 @@ import Data.Profunctor
 import Data.Profunctor.Unsafe
 import Control.Lens.Internal.Coerce
 
-import Data.ByteString       as StrictB
-import Data.ByteString.Lazy  as LazyB
-import Data.List.NonEmpty    as NonEmpty
-import Data.Text             as StrictT
-import Data.Text.Lazy        as LazyT
-import Data.Vector           as Vector
-import Data.Vector.Primitive as Prim
-import Data.Vector.Storable  as Storable
-import Data.Vector.Unboxed   as Unbox
-import Data.Sequence         as Seq
+import Data.ByteString                 as StrictB
```

Comment:
FWIW, I'd import all of these qualified. But even without this is :+1:

Reply:
True, there's even more suspicious imports besides the ones from `vector`. Do you think it's worth addressing all of them here, or would that be better suited for separate PRs?

Reply:
I decided to push a series of additional commits that change other commonly-clashing imports to be qualified.